content/en/blog/releases/v0.6.0.md

```diff
@@ -0,0 +1,78 @@
+
+But, not in all scenarios, it is possible to define all possible parameters upfront in a build strategy. For those tools that control behavior through environment variables, users can now define environment variables in their builds and build runs. Environment variable values can be referenced from Kubernetes secrets to support scenarios where for example tokens for private dependency management systems are needed. Examples of builds and build runs with environment variables can be found in our [build](https://github.com/shipwright-io/build/blob/v0.6.0/docs/build.md) and [build run](https://github.com/shipwright-io/build/blob/v0.6.0/docs/buildrun.md) documentation.
+
+## CLI
+
+Being based on Kubernetes, all our objects are Kubernetes custom resources with their ecosystem-specific syntax and concepts. But, not all our users should require a black belt in using Kubernetes. We are therefore proud to release the first version of our CLI. Build users can use it to easily manage their builds and build runs. Some highlights:
+
+Assuming you are connected to your Kubernetes cluster and have the credentials for your container registry in place, then creating a build for our go-sample is a single command:
+
+```sh
+$ shp build create sample-go --source-url https://github.com/shipwright-io/sample-go --source-context-dir source-build --output-image <CONTAINER_REGISTRY_LOCATION>/sample-go --output-credentials-secret <CONTAINER_REGISTRY_SECRET>
+Created build "sample-go"
+```
+
+And then it is just another single command to trigger a run and and to follow its logs. Nice. 😊
+
+```sh
+$ shp build run sample-go --follow
+Pod 'sample-go-9hbpj-l9t5j-pod-fz2sp' is in state "Pending"...
+Pod 'sample-go-9hbpj-l9t5j-pod-fz2sp' is in state "Pending"...
+[source-default] 2021/10/08 11:43:33 Info: ssh (/usr/bin/ssh): OpenSSH_8.0p1, OpenSSL 1.1.1g FIPS  21 Apr 2020
+[source-default] 2021/10/08 11:43:33 Info: git (/usr/bin/git): git version 2.27.0
+[source-default] 2021/10/08 11:43:33 Info: git-lfs (/usr/bin/git-lfs): git-lfs/2.11.0 (GitHub; linux arm64; go 1.14.4)
+[source-default] 2021/10/08 11:43:33 /usr/bin/git clone -h
+[source-default] 2021/10/08 11:43:33 /usr/bin/git clone --quiet --no-tags --single-branch --depth 1 -- https://github.com/shipwright-io/sample-go /workspace/source
+...
+```
+
```

Comment:
Should we include an example of using the `shp buildrun create` command here also?